collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesFragment.java

```diff
@@ -0,0 +1,402 @@
+package org.odk.collect.android.preferences;
+
+import android.accounts.Account;
+import android.accounts.AccountManager;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.CheckBoxPreference;
+import android.preference.EditTextPreference;
+import android.preference.ListPreference;
+import android.preference.Preference;
+import android.preference.PreferenceFragment;
+import android.preference.PreferenceManager;
+import android.preference.PreferenceScreen;
+import android.text.InputFilter;
+import android.text.Spanned;
+import android.util.Log;
+
+import com.google.android.gms.analytics.GoogleAnalytics;
+
+import org.odk.collect.android.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.odk.collect.android.preferences.PreferenceKeys.ARRAY_INDEX_GOOGLE_MAPS;
```

Comment:
In the case of static constant imports like these, I think it's better to do a single wildcard import.